util/color.zig

```diff
@@ -23,4 +23,9 @@ pub const Color = enum {
             .none => "",
         };
     }
+
+    // Method to return ANSI escape code for bold text.
+    pub fn bold() []const u8 {
```

Comment:
It's unfortunate that bold differs from the other codes, can it not be another enum value?

Reply:
True, good point since we already have reset as well there 👍